RetsTalk/RetsTalk/Chat/View/ChatView.swift

```diff
@@ -12,107 +12,95 @@ protocol ChatViewDelegate: AnyObject {
     func sendMessage(_ chatView: ChatView, with text: String)
 }
 
-@MainActor
-final class ChatView: UIView {
-    private let chattingTableView = UITableView()
+final class ChatView: BaseView {
+    
+    // MARK: Subviews
+    
+    private let chatTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.scrollsToTop = false
+        tableView.separatorStyle = .none
+        tableView.backgroundColor = .backgroundMain
+        tableView.allowsSelection = false
+        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "MessageCell")
```

Comment:
전역 상수로 만들어뒀습니다! 다 교체한다고 했는데 레거시가 남아있었나 보네요 ㅠㅠ
```suggestion
        tableView.register(UITableViewCell.self, forCellReuseIdentifier: "Constants.Texts.messageCellIdentifier")
```

Reply:
@MoonGoon72 `Suggested change` 상수에 따옴표가 들어갔어요ㅠ